x-pack/plugin/ilm/qa/multi-cluster/src/test/java/org/elasticsearch/xpack/indexlifecycle/CCRIndexLifecycleIT.java

```diff
@@ -278,7 +278,6 @@ public void testCcrAndIlmWithRollover() throws Exception {
         }
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/37165")
```

Comment:
The reason that we can remove this awaits fix is because the test is asserting that the shrink action is performed on the follower. The shrink action is prevented from being performed on the follower is any shard history retention leases exist on the follower with source `ccr`. The reason that these leases exist on the follower is because we copied them over from the commit point during recovery from remote! With this PR, this will no longer happen and this test can pass. 😌